_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: No more KubeVirt
```

Comment:
I'm not sure where this actually appears, but how about "Cockpit with several crash fixes and dropped KubeVirt support"?

Reply:
Agreed.

The summary:

1. appears at least in the tooltip when you hover the new badge on the front page
2. appears in a search
3. is used as keywords for searching (among many other data points)
4. _used to_ appear in /blog/ (but I replaced it with a list summary based on headlines)

Reply:
"Fixed crashers and dropped KubeVirt support"?